bindings_ffi/src/mls.rs

```diff
@@ -1157,25 +1157,11 @@ impl From<&FfiMetadataField> for MetadataField {
 
 #[uniffi::export(async_runtime = "tokio")]
 impl FfiConversations {
-    pub async fn create_group(
+    pub fn create_group_optimistic(
```

Comment:
One thing I did think about is this won't work for dms. Anyway we could make this work for dms? 🤔